src/test/java/net/sf/jabref/logic/util/version/VersionTest.java

```diff
@@ -13,8 +13,29 @@
 public class VersionTest {
 
     @Test
-    public void unknownVersion() {
-        Version version = new Version(BuildInfo.UNKNOWN_VERSION);
+    public void unknownVersionAsString() {
+        String versionText = BuildInfo.UNKNOWN_VERSION;
```

Comment:
@koppor No need to extract the parameter into a variable `versionText` in these tests, right?